components/org.wso2.carbon.status.dashboard.web/src/distributedView/AppView.jsx

```diff
@@ -0,0 +1,352 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import React from "react";
+import {Link} from "react-router-dom";
+import SyntaxHighlighter from "react-syntax-highlighter";
+
+//App Components
+import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
+import Header from "../common/Header";
+import ParentAppTable from "./ParentAppTable";
+//Material UI
+import HomeButton from "material-ui/svg-icons/action/home";
+import {
+    Card,
+    CardHeader,
+    CardText,
+    Dialog,
+    Divider,
+    FlatButton,
+    Snackbar, RaisedButton
+} from "material-ui";
```

Comment:
Don't use `"` for strings, always use `'`. Please fix in other places where you have used `"`.

Reply:
this is due to JS code style guideline